### ISSUE TITLE
Check for Element type?

### ISSUE THREAD

**Issue by cdukes** [eduardoboucas/include-media-export#11]
Hi,

We log JS errors from our users, and very infrequently, this line in `include-media.js` causes some errors:

`if (window.getComputedStyle && (window.getComputedStyle(element, '::after').content !== ''))`

For instance, Googlebot encountered `Failed to execute 'getComputedStyle' on 'Window': parameter 1 is not of type 'Element'.` today

And 48 Firefox users (versions 47-50) encountered `TypeError: window.getComputedStyle(...) is null` over the past couple months

I can't pin down why this sometimes happens and have never been able to reproduce it myself, but I think it would help clear out some edge cases to check that window.getComputedStyle is a function and element is an Element before trying to use them?

Thanks!

**Comment by eduardoboucas** [comment]
I was able to replicate that error by including the script in the `<head>`, where the DOM isn't available yet. Could this be the case with your errors too?

In any case, it sounds like a good idea to add a safeguard around those expressions.

Thanks for raising the issue!

**Comment by mpdude** [comment]
Maybe #12 solves this as well?

**Comment by mpdude** [comment]
Re-read the problem description – #12 probably does *not* help: The exceptions occur outside the current try/catch block.

Would it make sense to have a much broader exception handling in that method, catching all sorts of errors and reverting the `breakpoints` variable to `false`?

**Comment by eduardoboucas** [comment]
> Would it make sense to have a much broader exception handling in that method, catching all sorts of errors and reverting the breakpoints variable to false?

Yes, that sounds sensible. I don't have the capacity to look into this right now, so I'll leave it open for when I or someone else has the chance to make it happen.

Thanks!
